Decent.Minecraft.Client/Java/JavaBlock.cs

```diff
@@ -80,7 +80,9 @@ static JavaBlock()
                 d == 1 ? new MossyStoneBricks() :
                 d == 2 ? new CrackedStoneBricks() :
                 (StoneBricks)new ChiseledStoneBricks();
-            _ctors[Id<Wood>()] = d => new Wood((WoodSpecies)(d & 0x3), (Axis)(d & 0xC));
+            // Wood is both 17 and 162 depending on species
+            _ctors[17] = d => new Wood((WoodSpecies)(d & 0x3), (Axis)(d & 0xC));
```

Comment:
Look at the precedent of `Snow` and `SnowLayer`: the main one, `Snow`, is the only one that has an actual type on our side, and it's in the Java registry. The other one, `SnowLayer`, is defined as a constant so we don't have a magic number hanging in the serialization code. I think this is pretty much analogous to what you're trying to do with wood.